<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>347</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <widget class="QDialogButtonBox" name="btnBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>381</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="grpBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>591</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Options</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>41</y>
      <width>543</width>
      <height>132</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblInterfaceLanguage">
       <property name="text">
        <string>Interface Language:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="cmbBoxInterfaceLanguage"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblFontAndSize">
       <property name="text">
        <string>Font &amp; Size</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QFontComboBox" name="fcmbBoxFontFamily"/>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spinBoxFontSize"/>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="chkBoxEnableLoging">
       <property name="text">
        <string>Enable Logging</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QCheckBox" name="chkBoxRestoreWindowSize">
       <property name="text">
        <string>Restore original window size</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
